docs/user_guide/gui/operations_window.rst

```diff
@@ -29,15 +29,9 @@ and after** the filter was applied.
 
 There are several tickboxes to customise the behaviour of the preview area.
 
-Legend
```

Comment:
Is it worth updating the screenshot on the Operations Window page as well, as I think it's still showing these checkboxes at the moment?